### PR TITLE
hotfix | lib.nix:  update access (& refactor slightly for more clarity)

### PR DESCRIPTION
**Pull request by deepfire**

### CODE REVIEW

File: lib.nix [hunk 1]
```diff
@@ -51,26 +51,29 @@ in lib // (rec {
...
   alexandersKey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDUGnVIJ5w4OP9RaTZKM929Wt69UyvbBLyK65FNAkNXraKJ4t7rp41AaiY17byBET5vHxiF7CdVxi2nng6SQKp8VOlCPvF/AN4ZjruFDIXunJLERzf7GCR+bQPu4E/uj1hR9lIs8/jRKtHN6cZ7jcRni8b7BSIISInHTDwGm7XhdpRQlrxMK67oMXQuuOnEEShmH+ER0ICGk/o33SIAK2bebfRgCdI7N8iMZ/vw9ACHptN1wG70H2OVZADFY7J2mS0EFz+iFx5aOn/EbQfgNb8y9+K4+kfWxl/lwFw0Lj6YwUMEvqlqtHRYKq7uLT18JzYfM4mtGkYaNGDmzfqqpq5UTIW0txr9luNf5CRZeJctsY/OpxV1dPZXIMMENl1Y3a2GnD2YfKZ04eQrUgFN4R5J9S3dhkqUNlzc72/dkbCbWVk86+uG4k+DAm5MCua9xaF7jEuwd/BnCw72ce06hf5soyiIpOLhkuMyTY6/rXiSYETdGlJKTgx1S3bqa5FD4yXA++fwxoOLww/ZMRRdiufSk3KJBvAQMP0lKxR7uQA9S4ZGzLifWII21f2x1mLvknzzhKY4hZLmXA/TIeeygWYmIF5ldoTVtWP6duLspFzC3Fei+tm6ZzYcuuAX6jaT76Kkrsy9bQOjuI/mwwILzD6qZeu41Y/FsvQzgVqJ91HX/w== codieplusplus@apax.net";
 
-  devKeys = [ domenKey alanKey jakeKey dshevchenkoKey larsKey philippKey kosergeKey ksaric sectore michaelKey akegaljKey alfredoKey rodneyKey vasilisKey andreasKey samKey alexandersKey ];
+  devKeys = devOpsKeys ++ [ alanKey dshevchenkoKey larsKey philippKey ksaric akegaljKey alfredoKey vasilisKey andreasKey alexandersKey ];
```
Comment (rvl) [inline lib.nix]: i think put list items one per line and sort them.
We previously had merge conflicts on these lines which were more difficult than necessary to resolve.

Comment (deepfire) [reply lib.nix]: yeah, did that actually : -)

**Comment by deepfire** [comment]
@rvl, please, if you have questions, it would help to formulate them as such.  As it stands you're now blocking acceptance of the PR -- and for an arguably very minor reason.

As to your question, the historic practice is to keep the list sorted by addition date.

**Comment by deepfire** [comment]
@jmitchell, done!

**Comment by jmitchell** [comment]
We agreed to have each section of keys alphabetically sorted.

**Pull request merged**